py3status/modules/yubikey.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+"""
+Show an indicator when YubiKey is waiting for a touch.
+
+Configuration parameters:
+    format: Display format for the module.
+        (default '\?if=is_waiting YubiKey')
+    gpg_check_timeout: Time to wait for gpg check response.
+        Use value as small as possible that doesn't introduce false positives.
+        (default 0.1)
```

Comment:
(Q) Is it useful for users to have `gpg_check_timeout` config if the default is `0.1`? The message also say `Use value as small as possible that doesn't introduce false positives`. Another thing... I see a hardcoded `time.sleep(0.25)` too... right after that config too... so what can they do with this config?